doc/modules/classification_threshold.rst

```diff
@@ -1,6 +1,6 @@
 .. currentmodule:: sklearn.model_selection
 
-.. _TunedThresholdClassifierCV:
+.. _threshold_tunning:
```

Comment:
I referenced this page starting here for `decision_threshold_curve` because I thought this first paragraph was appropriate, for context, not just the section I added. Not 100% on this though and happy to change